mlir/include/mlir/Dialect/Rock/utility/loweringUtils.h

```diff
@@ -22,6 +22,18 @@ namespace rock {
 struct ConvolutionDims;
 struct GemmSize;
 
+struct GPUViews {
+  ArrayAttr gridwiseView;
+  ArrayAttr blockwiseView;
```

Comment:
Can we get some comments here on what these views actually are?

Reply:
I've added for these and other here.